### DISCUSSION TITLE
"Make sure steam is running"

### DISCUSSION THREAD

**Discussion by Nighteg** [X2CommunityCore/xcom2-launcher#282] [Help] [answered]
Steam IS RUNNING! Why do i get this message when I'm running the launcher? 
Moreover, I was told that I don't need steam to work with AML..

**Comment by RevZero** [comment, marked as answer]
https://github.com/X2CommunityCore/xcom2-launcher/wiki/FAQ#is-there-a-way-to-use-aml-without-steam

https://github.com/X2CommunityCore/xcom2-launcher/wiki/Troubleshooting#dialog-please-start-steam-first-appears-
